sinks/api/supported_metrics.go

```diff
@@ -109,26 +124,41 @@ var statMetrics = []SupportedStatMetric{
 			ValueType:   ValueInt64,
 			Units:       UnitsBytes,
 		},
-		HasValue: func(spec *cadvisor.ContainerSpec) bool {
+		HasValue: func(spec *source_api.ContainerSpec) bool {
 			return spec.HasMemory && (spec.Memory.Limit > 0)
 		},
-		GetValue: func(spec *cadvisor.ContainerSpec, stat *cadvisor.ContainerStats) []InternalPoint {
+		GetValue: func(spec *source_api.ContainerSpec, stat *source_api.ContainerStats) []InternalPoint {
 			return []InternalPoint{{Value: int64(spec.Memory.Limit)}}
 		},
 		OnlyExportIfChanged: true,
 	},
 	{
 		MetricDescriptor: MetricDescriptor{
+			Name:        "memory/request",
+			Description: "Memory request",
```

Comment:
Should we clarify request vs limit? If we were to run with cadvisor in standalone mode, what will this metric be?